integration/nwo/standard_networks.go

```diff
@@ -227,6 +227,24 @@ func MinimalRaft() *Config {
 	return config
 }
 
+// Like FullSolo, FullEtcdRaft is a configuration with two organizations and two peers per org.
+func FullEtcdRaft() *Config {
+	config := FullSolo()
```

Comment:
Isn't it a bit awkward to still call the solo constructor? 
Why not just call `MinimalRaft()` and change it accordingly?